### PR TITLE
RHMAP-17191 update about-admin-guide

### PR DESCRIPTION
**Pull request by pmacko1**
decluttering https://github.com/aerogear/aerogear.org/pull/683
https://issues.jboss.org/browse/RHMAP-17191

### CODE REVIEW

File: docs/digger/admin/about-admin-guide.asciidoc [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
+The following text describes optional post-installation steps:
 
-The following text serves as a reference for AeroGear Digger components administration:
-
-- Nagios monitoring
-- Installing and configuring Jenkins Plugins
-- Using HTTP proxy for Digger installation and ...
-- 
\ No newline at end of file
+- Installing Jenkins Plugins
```
Comment (laurafitzgerald) [inline docs/digger/admin/about-admin-guide.asciidoc]: @pmacko1 why did the sections change here?

Comment (pmacko1) [reply docs/digger/admin/about-admin-guide.asciidoc]: @laurafitzgerald to reflect the changes to the guide
these mr's are just splinters from RHMAP-18330, checkout that branch to see how the whole repo looks with all changes

Comment (pmacko1) [reply docs/digger/admin/about-admin-guide.asciidoc]: It is just a list of components that can be "administrated" after installation. Actually I was thinking about removing the administration guide altogether

Comment (laurafitzgerald) [reply docs/digger/admin/about-admin-guide.asciidoc]: @pmacko1 what is the motivation behind removing the http-proxy one? Is the section about it also removed from the admin guide? removing the guide would be a separate JIRA.

Comment (pmacko1) [reply docs/digger/admin/about-admin-guide.asciidoc]: @laurafitzgerald use-http-proxy was moved to installation guide.

Comment (laurafitzgerald) [reply docs/digger/admin/about-admin-guide.asciidoc]: @pmacko1 okay thanks for the clarification. feel free to merge this one.

**Pull request merged**